earth2mip/inference_ensemble.py

```diff
@@ -252,10 +252,13 @@ def perturb(x, rank, batch_id, device):
         if rank == 0 and batch_id == 0:  # first ens-member is deterministic
             noise[0, :, :, :, :] = 0
 
-        scale = torch.tensor(
-            [channel_stds[channel] for channel in model.in_channel_names],
-            device=x.device,
-        )
+        scale = []
```

Comment:
Please add a comment here discussing what this is doing e.g.:

`# for channels that are not present in normalization dictionary, perturbation will be skipped by default`

For those looking in the code some more